_posts/2015-09-15-rnd-day-2015.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: "R&D Day 2015 in the spotlight"
+date: "2015-09-15 08:10:00"
+icon: search-plus
+categories: tech-stories
+authors: ["Birgit"]
+---
+
+ePages developers from all company locations convened in Hamburg to spend an amazing time together on software development topics!
+It was all about interesting lectures and workshops, healthy discussions and knowledge transfer among different teams.
+
+Curtains up for these fantastic topics:
+
+# Spotlight: ePages software architecture
+
+Team Black started the R&D Day presenting their approach on a new software architecture that should be the technical foundation for ePages in the years to come.
```

Comment:
Do you think you need to write what R&D is in full first? Probably not its self explanatory right?